app/views/controllers/projects/show.html.erb

```diff
@@ -1,11 +1,18 @@
 <%
 add_project_banner(@project)
 @container = :wide
+violations_count = @project.count_violations
+violations_button_class =
+  if violations_count.zero?
+    "btn btn-default btn-lg"
+  else
+    "btn btn-warning btn-lg"
+  end
```

Comment:
@nimmolo See [earlier question about styling this button](https://github.com/MushroomObserver/mushroom-observer/pull/2021/files#r1519139794).

Reply:
This seems right, though, to me. What's happening with it at smaller screens that's not good?

(Sorry I have migrations on my UserStats branch that are making it kind of involved to switch branches right now. Should be done soon!)